core/deployment/src/main/java/io/quarkus/deployment/util/WebJarUtil.java

```diff
@@ -44,7 +44,10 @@
 
 /**
  * Utility for Web resource related operations
+ * 
+ * @deprecated Use WebJarBuildItem and WebJarResultBuildItem instead.
```

Comment:
If you rename `WebJarResultBuildItem`, make sure this is done everywhere including here.